README.md

```diff
@@ -112,6 +111,45 @@ query {
   }
 }
 ```
+## Testing Guidelines
+When writing tests, try to follow these guidelines if possible:
+
++ Tests should be as readable as possible and not complex at all. You should understand them by looking at them just once.
++ Local helper functions defined in test files should have functional and easy-to-understand rather than technical names. Meaning, `clickNewUserButton()` is better than `clickElementByTypeAndTestId('button','new-user-button')`.
++ More general use helpers like 'clickElementByTypeAndTestId' can be used within the local helper functions if preferred. The reason for functional naming preference lies in increased readability of tests.
++ Avoid any duplication of helper functions across several files! If using the same functions in several tests (files), there's a tendency to copy-paste the whole file and then rewrite tests. This leads to code duplication of helper functions. Instead, helper functions needed in several locations should be defined in one place should be available globally. Find the matching one by name or create a new one. Avoid creating miscellaneuos file names as it tends to lead to chaos.
++ Current codebase doesn't 100% follow everything stated above but it'd definitely help organising the test helpers accordingly from now on.
+
+![Selection_599](https://user-images.githubusercontent.com/8964422/77221481-6a190d00-6b4a-11ea-88d7-9fc70ce1c982.png)
+
+## Architecture overview
+
+All our architecture decisions are logged in ADRs which you can find [here](docs/adr/adr_template.md).
+Here, is a list of intro tutorials for each technologies / frameworks / languages below.
+
+#### Frontend
+
+- [Typescript](https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes.html)
+- [Custom Rebass Library](https://github.com/boxwise/react-components)
+- [React](https://reactjs.org/docs/getting-started.html)
+- [Apollo](https://www.apollographql.com/docs/react/)
+
+#### Backend
+
+- [GraphQL](https://graphql.org/learn/)
+- [Ariadne](https://ariadnegraphql.org/docs/flask-integration.html)
+- [Flask](https://flask.palletsprojects.com/en/1.1.x/tutorial/layout/)
+- [PeeWee](http://docs.peewee-orm.com/en/latest/peewee/quickstart.html)
+- MySQL
+
+#### Authentication
+
+- [Auth0](https://auth0.com/docs/quickstart/spa/react)
+
+#### Testing
+- [React Testing Library / Jest](https://www.freecodecamp.org/news/8-simple-steps-to-start-testing-react-apps-using-react-testing-library-and-jest/#2-testing-dom-elements)
```

Comment:
if we are going to include resources, we should make sure that they are high quality ones. Maybe @jamescrowley can check it out and see if one of these is a good one, or if he can suggest something better? ;)

https://www.freecodecamp.org/news/8-simple-steps-to-start-testing-react-apps-using-react-testing-library-and-jest/#2-testing-dom-elements
https://medium.com/@boyney123/my-experience-moving-from-enzyme-to-react-testing-library-5ac65d992ce

Reply:
no preferences. I just tried to put links to tutorials to get quickly a hang of it. I went through the freecodecamp tutorial and liked it better than the starting guides from React-Testing-Library, which did not touch much.

Reply:
I'm happy to resolve this if we add the tip for people moving from Enzyme.